sql/core/src/main/java/org/apache/spark/sql/execution/vectorized/OnHeapColumnVector.java

```diff
@@ -366,6 +396,12 @@ public double getDouble(int rowId) {
     }
   }
 
+  @Override
+  public void getDoubles(double[] dst, int rowId, int count) {
+    assert(dictionary == null);
+    System.arraycopy(byteData, rowId, dst, 0, count);
+  }
+
```

Comment:
@dongjoon-hyun for example, [these lines](https://github.com/apache/spark/blob/master/sql/core/src/test/scala/org/apache/spark/sql/execution/vectorized/ColumnarBatchSuite.scala#L763-L766) uses this new API. Would it be better to add additional tests?